src/ensembl/src/content/app/species/state/sidebar/speciesSidebarSelectors.ts

```diff
@@ -14,7 +14,18 @@
  * limitations under the License.
  */
 
+import { getActiveGenomeId } from 'src/content/app/species/state/general/speciesGeneralSelectors';
 import { RootState } from 'src/store';
 
 export const isSidebarOpen = (state: RootState) =>
   state.speciesPage.sidebar.isOpen;
+
+export const getActiveGenomeSidebarPayload = (state: RootState) => {
+  const activeGenomeId = getActiveGenomeId(state);
+
+  if (!activeGenomeId) {
+    return;
```

Comment:
this makes it return `undefined`. Probably better return null for consistency with the next line?

Reply:
That's done